README.md

```diff
@@ -61,35 +63,32 @@ Basically all settings defined in the Configuration File Options documentation p
 
 ### Cookbook specific attributes
 
-* `node[:mongodb][:reload_action]` - Action to take when MongoDB conf files are
-    modified, default is `"restart"`
-* `node[:mongodb][:package_version]` - Version of the MongoDB package to install, default is nil
-* `node[:mongodb][:client_role]` - Role identifying all external clients which should have access to a mongod instance
+* `node['mongodb']['reload_action']` - Action to take when MongoDB conf files are modified, default is `"restart"`
+* `node['mongodb']['package_version']` - Version of the MongoDB package to install, default is nil
+* `node['mongodb']['client_role']` - Role identifying all external clients which should have access to a mongod instance
```

Comment:
I think the name of the attribute here is wrong. This should be `'client_roles'`. But it is not a fail of this PR.